scripts/docker/entrypoint_ci.sh

```diff
@@ -239,6 +239,19 @@ function check_download_sqlalchemy() {
     pip check
 }
 
+# Download minimum supported version of pendulum to run tests with it
+function check_download_pendulum() {
+    if [[ ${DOWNGRADE_PENDULUM=} != "true" ]]; then
+        return
+    fi
+    min_pendulum_version=$(grep "\"pendulum>=" pyproject.toml | sed "s/.*>=\([0-9\.]*\).*/\1/" | xargs)
```

Comment:
❤️ it

Reply:
BTW. This is EXACTLY what I wanted to do for ALL dependencies of ours in https://github.com/apache/airflow/issues/35549

Reply:
Feel free to resolve and merge @Taragolis :D

Reply:
At least this step help to found that Airflow can't work with pendulum 2.0.0 (previous min version) and had some issues.